### PR TITLE
Can now export Players and PlayerLists as CSV and JSON.

### PR DESCRIPTION
**Pull request by GoldenBadger**

### CODE REVIEW

File: lib/libarbiter/player.cc [hunk 1]
```diff
@@ -64,4 +64,22 @@ Player Player::FromJSON(std::string player_string) {
   return Player(player_name, player_elo);
 }
 
+std::string Player::ExportCSV() const {
+  // The name field might include quotes, so we must escape them with another
+  // quote.
```
Comment (VoyTechnology) [inline lib/libarbiter/player.cc:69]: Is there no other "proper" way of escaping string? How would this work for other special characters which are not allowed?

File: lib/libarbiter/player.h [hunk 2]
```diff
@@ -45,6 +46,18 @@ class Player {
...
+  /**
+   *  Serialises the Player to a JSON object.
+   *
+   *  The JSON is stored in a boost::property_tree::ptree.
+   */
+  boost::property_tree::ptree ExportJSON() const;
+
   boost::uuids::uuid id() const { return id_; }
   std::string name() const { return name_; }
   int elo() const { return elo_; }
```
Comment (VoyTechnology) [inline lib/libarbiter/player.h:63]: what is `elo` again?

File: lib/libarbiter/player.cc (hunk 1 above)
Comment (GoldenBadger) [inline lib/libarbiter/player.cc:69]: This is the proper way to escape quotes in CSV. https://tools.ietf.org/html/rfc4180#page-2

File: lib/libarbiter/player.h (hunk 2 above)
Comment (GoldenBadger) [inline lib/libarbiter/player.h:63]: A system for rating the relative skill levels of chess players. https://en.wikipedia.org/wiki/Elo_rating_system

File: lib/libarbiter/player.cc (hunk 1 above)
Comment (VoyTechnology) [inline lib/libarbiter/player.cc:69]: I mean like a library that handles that sort of stuff together with any other special characters that might occur

Comment (GoldenBadger) [inline lib/libarbiter/player.cc:69]: I haven't found any decent ones. The standard says that this is sufficient.

**Comment by VoyTechnology** [comment]
LGTM

**Pull request merged**